### PR TITLE
Improving docs about test script creation

### PR DESCRIPTION
**Pull request by shan100github**
##### Checklist

- [x] documentation is changed or added (if applicable) 

### Description of change
Adding in details about "Goss test script creation" methods and manul edit scenario.
Currently, no specific section covering details about test script generation.

### CODE REVIEW

File: docs/manual.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
     * [render, r \- Render gossfile after importing all referenced gossfiles](#render-r---render-gossfile-after-importing-all-referenced-gossfiles)
     * [serve, s \- Serve a health endpoint](#serve-s---serve-a-health-endpoint)
     * [validate, v \- Validate the system](#validate-v---validate-the-system)
+* [Goss test script creation](#goss-test-script-creation)
```
Comment (pedroMMM) [inline docs/manual.md]:
```suggestion
* [Goss test creation](#goss-test-creation)
```

File: docs/manual.md [hunk 2]
```diff
@@ -322,6 +323,29 @@ $ echo $?
 2
 ```
 
+## Goss test script creation
+Goss test scripts can be created by using either of following methods.
+1. goss autoadd <resource to test>
+2. goss add <resource to test> 
+3. manually create YAML test script by hand.
+
+To customize the parameters generated by `goss add` and `goss autoadd` YAML scripts we need to manually edit it.
```
Comment (pedroMMM) [inline docs/manual.md]:
```suggestion
## Goss test creation
Goss tests can be created by using either of following methods.
1. goss autoadd <resource to test>
2. goss add <resource to test> 
3. manually create YAML/JSON test file by hand.

To customize the parameters generated by `goss add` and `goss autoadd` YAML scripts you need to manually edit it.
```

File: docs/manual.md [hunk 3]
```diff
@@ -322,6 +323,29 @@ $ echo $?
...
+
+`goss add package nginx` will generate below YAML
+```
+package:
+  nginx:
+    installed: true
+    versions:
+    - 1.17.8
+```
+to test uninstall scenario we need to manually edit it and set it as below.
```
Comment (pedroMMM) [inline docs/manual.md]:
```suggestion
To test uninstall scenario you would need to manually edit it and set it as below.
```

**Comment by shan100github** [comment]
Thanks for your feedback @pedroMMM

File: docs/manual.md [hunk 4]
```diff
@@ -322,6 +323,29 @@ $ echo $?
 2
 ```
 
+## Goss test creation
+Goss tests can be created by using either of following methods.
+1. goss autoadd <resource to test>
+2. goss add <resource to test> 
+3. manually create YAML/JSON test file by hand.
+
+To customize the parameters generated by `goss add` and `goss autoadd` YAML scripts you need to manually edit it.
```
Comment (pedroMMM) [inline docs/manual.md]:
```suggestion
To customize the parameters generated by `goss add` and `goss autoadd` YAML file you need to manually edit it.
```

**Comment by shan100github** [comment]
@aelsabbahy @pedroMMM 
hope looks good to merge.

**Comment by shan100github** [comment]
thanks for approval @pedroMMM even I overlooked it :(

**Pull request merged**